GuildedRoseLLC/ItemsViewController.swift

```diff
@@ -10,14 +10,20 @@ public class ItemsViewController: UIViewController, UICollectionViewDelegate {
         
     override public func viewDidLoad() {
         super.viewDidLoad()
-        let items = Item.getItems()
-        configureDataSource(items: items)
-        toggleListDisplay(items: items)
+        dataSource = ItemCollectionViewDataSource()
+        configureDataSource(items: [])
         
         greeting.accessibilityIdentifier = "Greeting"
         itemCollectionView.accessibilityIdentifier = "itemCollectionView"
     }
     
+    override public func viewWillAppear(_ animated: Bool) {
```

Comment:
I would have expected to see the tests updated alongside these changes. In theory, test driving `viewWillAppear` should be very similar to test driving `viewDidLoad`

Reply:
Will be part of PR resubmission. ♻️